### PR TITLE
Add a monthly nuking service

### LINKED ISSUE #40: Prune images built by steps
We’re not removing images built by a step when `STEP_DOCKER_FILE` is set. We should extend the `docker-volume-prune` service to also remove these images.

### PR DESCRIPTION
**Pull request by FintanH**
Fixes #40 

This is my attempt to add the image removal service. I mostly cargo-culted so please let me know if I did anything wrong :)

### CODE REVIEW

File: ci/linux/etc/systemd/system/docker-volume-nuke.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+# Get all docker image IDs and remove them.
+define -r images = $(docker images --format='{{.ID}}')
```
Comment (kim) [inline ci/linux/etc/systemd/system/docker-volume-nuke.sh]: This does not look like valid bash :)

Comment (kim) [reply ci/linux/etc/systemd/system/docker-volume-nuke.sh]:
```suggestion
images="$(docker images --format='{{.ID}}')"
```

Comment (FintanH) [reply ci/linux/etc/systemd/system/docker-volume-nuke.sh]: Huh, my vim syntax checker didn't give out :sweat_smile: Thanks!

**Comment by kim** [comment]
Yeah, uhm, sorry, it seems like the `docker-system-prune` service is actually supposed to remove images. And it actually works, it's just that the images will be pulled down again very quickly.

Sorry for the confusion!

**Comment by FintanH** [comment]
So is there an actual issue that's different to what we thought?

**Comment by FintanH** [comment]
Nevermind, just saw the comment in #40

**Pull request closed without merging**